packages/filebrowser-extension/src/index.ts

```diff
@@ -313,21 +312,53 @@ const defaultFileBrowser: JupyterFrontEndPlugin<IDefaultFileBrowser> = {
   description: 'Provides the default file browser',
   provides: IDefaultFileBrowser,
   requires: [IFileBrowserFactory],
-  optional: [IRouter, JupyterFrontEnd.ITreeResolver, ILabShell],
+  optional: [IRouter, JupyterFrontEnd.ITreeResolver, ILabShell, ITranslator],
   activate: async (
     app: JupyterFrontEnd,
     fileBrowserFactory: IFileBrowserFactory,
     router: IRouter | null,
     tree: JupyterFrontEnd.ITreeResolver | null,
-    labShell: ILabShell | null
+    labShell: ILabShell | null,
+    translator: ITranslator
   ): Promise<IDefaultFileBrowser> => {
     const { commands } = app;
+    const trans = translator.load('jupyterlab');
```

Comment:
```suggestion
    const trans = (translator ?? nullTranslator).load('jupyterlab');
```

`nullTranslator` needs to be imported from `@jupyterlab/translation`

Reply:
@krassowski Thanks for your suggestions. I indeed forgot to properly treat the possibility of not having a translator.